lighthouse-core/scripts/pptr-run-devtools.js

```diff
@@ -228,6 +295,12 @@ async function run() {
   if (customDevtools) {
     console.log(`Using custom devtools frontend: ${customDevtools}`);
     console.log('Make sure it has been built recently!');
+    if (!customDevtools.startsWith('file://')) {
+      throw new Error('custom-devtools-frontend must be a file:// URL');
+    }
+    if (!fs.existsSync(fileURLToPath(customDevtools))) {
```

Comment:
Running locally on mac I get an error here:

```
(node:43463) UnhandledPromiseRejectionWarning: TypeError [ERR_INVALID_FILE_URL_HOST]: File URL host must be "localhost" or empty on darwin
    at new NodeError (internal/errors.js:322:7)
    at getPathFromURLPosix (internal/url.js:1350:11)
    at fileURLToPath (internal/url.js:1373:50)
    at run (file:///Users/asraine/lighthouse/lighthouse-core/scripts/pptr-run-devtools.js:301:24)
    at file:///Users/asraine/lighthouse/lighthouse-core/scripts/pptr-run-devtools.js:353:1
    at ModuleJob.run (internal/modules/esm/module_job.js:183:25)
    at async Loader.import (internal/modules/esm/loader.js:178:24)
    at async Object.loadESM (internal/process/esm_loader.js:68:5)
```
Am I missing something, tried on Node 14 & 16.

Reply:
Works for me.

```
yarn run-devtools --custom-devtools-frontend=file://$PWD/.tmp/chromium-web-tests/devtools/devtools-frontend/out/Default/gen/front_end https://www.example.com
```

node 14.8.1

Reply:
I think it's because a relative path is set in `DEVTOOLS_PATH` if run locally.

Reply:
What is your command?

Reply:
`yarn smoke --runner devtools csp`